source/sfp/4_vegetation/sfp_408_nl_lsvi_stilwater/NEWS.md

```diff
@@ -0,0 +1,22 @@
+# Wijzigingen t.o.v. vorige versies
+
+## [2025.03](../2025.03/index.html)
+
+-   ...
```

Comment:
Deze ... moeten nog vervangen worden door een tekst. Als het de eerste versie is, kan je hier gewoon schrijven: "dit is de eerste versie van het protocol". Of als het een omzetting is van een bestaand protocol, kan je dat ook schrijven en er naar verwijzen.